llvm/test/CodeGen/X86/dag-large-offset.ll

```diff
@@ -0,0 +1,36 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=i386-pc-windows-msvc | FileCheck %s
+
+define dso_local noundef i32 @foo(i1 %b) local_unnamed_addr #0 {
```

Comment:
Can drop attributes and should comment the test

Reply:
Thanks, updated.